src/Map/assets/dist/abstract_map_controller.js

```diff
@@ -36,7 +36,10 @@ class default_1 extends Controller {
         return infoWindow;
     }
     dispatchEvent(name, payload = {}) {
-        this.dispatch(name, { prefix: 'ux:map', detail: payload });
+        this.dispatch(name, {
+            prefix: 'ux:map',
+            detail: Object.assign({}, this.additionalEventPayload, payload),
```

Comment:
```suggestion
            detail: { ...this.additionalEventPayload, ...payload },
```

?

Reply:
I was afraid of breaking param re-assignment, but that should be fine in fact